runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -7686,10 +7698,7 @@ int post_fma_ct(c_nodeid_t* locale_v, gni_post_descriptor_t* post_desc)
          pdc != NULL;
          pdc = pdc->next_descr, i++) {
       pdc->ep_hndl = cd->remote_eps[locale_v[i]];
-      if (post_desc->type == GNI_POST_FMA_PUT)
-        PERFSTATS_ADD(sent_bytes, pdc->length);
-      else
-        PERFSTATS_ADD(rcvd_bytes, pdc->length);
+      PERFSTATS_ADD_POST(post_desc);
```

Comment:
Shoot, this is wrong. I'm adding post_desc every time here instead of the current pdc -- will fix in an upcoming PR

Reply:
Fixed in https://github.com/chapel-lang/chapel/pull/10725